### PR TITLE
Update euclid.

### PR DESCRIPTION
**Pull request by emilio**

### CODE REVIEW

**Comment by RazrFalcon** [comment]
I've tried to do this yesterday, but `lyon_path` was still outdated.

**Comment by emilio** [comment]
@RazrFalcon yeah, it was updated today.

**Comment by emilio** [comment]
I guess you can also review this @RazrFalcon (if it looks sane to you)

**Comment by RazrFalcon** [comment]
Yep, I will merge this as soon as CI will be fixed.

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "font-kit"
-version = "0.3.1"
+version = "0.4.0"
```
Comment (RazrFalcon) [inline Cargo.toml]: I think we should keep it for a separate commit.

Comment (emilio) [reply Cargo.toml]: Why is that?

Comment (emilio) [reply Cargo.toml]: Well, I can split it in two anyhow.

**Comment by emilio** [comment]
Fixed the formatting that was making travis fail and a test too.

**Comment by emilio** [comment]
And pushed the version bump as a separate commit.

**Comment by emilio** [comment]
I spent a bit more time and fixed the Windows build.
